libc/startup/linux/aarch64/start.cpp

```diff
@@ -193,6 +193,8 @@ __attribute__((noinline)) static void do_start() {
   LIBC_NAMESPACE::main_thread_attrib.atexit_callback_mgr =
       LIBC_NAMESPACE::internal::get_thread_atexit_callback_mgr();
 
+  LIBC_NAMESPACE::atexit(
```

Comment:
please add a comment here (and in the other startup code) mentioning that this `atexit` call needs to be the first one.